get-started/the-stack.md

```diff
@@ -10,92 +11,86 @@ products:
 
-{{ls}} is a data collection engine with real-time pipelining capabilities. It can dynamically unify data from disparate sources and normalize the data into destinations of your choice. {{ls}} supports a broad array of input, filter, and output plugins, with many native codecs further simplifying the ingestion process. [Learn more about {{ls}}](logstash://reference/index.md).
+Learn more in [The {{es}} data store](/manage-data/data-store.md)
 
+### Visualize and query your data [kibana-navigation-search]
 
-### Store [_store]
+While {{es}} stores your data, **Kibana** is the user interface where you can explore, visualize, and manage it. It provides a window into your data, allowing you to quickly gain insights and understand trends.
 
-#### {{es}} [stack-components-elasticsearch]
+With Kibana, you can:
 
-{{es}} is the distributed search and analytics engine at the heart of the {{stack}}. It provides near real-time search and analytics for all types of data. Whether you have structured or unstructured text, numerical data, or geospatial data, {{es}} can efficiently store and index it in a way that supports fast searches. {{es}} provides a REST API that enables you to store data in {{es}} and retrieve it. The REST API also provides access to {{es}}'s search and analytics capabilities. [Learn more about {{es}}](/get-started/index.md).
+* Use **Discover** to interactively search and filter your raw data.  
+* Build custom **visualizations** like charts, graphs, and metrics with tools like **Lens**, which offers a drag-and-drop experience.  
+* Assemble your visualizations into interactive **dashboards** to get a comprehensive overview of your information.  
+* Analyze geospatial data using the powerful **Maps** application.
 
+At the same time, Kibana works as the user interface of all Elastic solutions, like Elastic Security and Elastic Observability, providing ways of configuring Elastic to suit your needs and offering interactive guidance.
 
-### Consume [_consume]
+A **query** is a question you ask about your data, and Elastic provides several powerful languages to do so. You can query data directly through the API or through the user interface in Kibana.
 
-Use {{kib}} to query and visualize the data that’s stored in {{es}}. Or, use the {{es}} clients to access data in {{es}} directly from common programming languages.
+* **Kibana Query Language (KQL)** is the text-based language used in the **Discover** search bar, perfect for interactive filtering and exploration.  
+* **Elasticsearch Query Language (ES|QL)** is a powerful, modern query language that uses a familiar pipe-based syntax to transform and aggregate your data at search time.  
+* **Event Query Language (EQL)** is a specialized language designed to query sequences of events, which is particularly useful for security analytics and threat hunting.
 
-#### {{kib}} [stack-components-kibana]
+Learn more in [Explore and analyze data with Kibana](/explore-analyze/index.md) 
```

Comment:
suggest always using empty titles so the system can populate them with the target page title as it is tweaked

```suggestion
Learn more in [](/explore-analyze/index.md) 
```

Reply:
TIL! This isn't documented in https://docs-v3-preview.elastic.dev/elastic/docs-builder/tree/main/syntax/links. I shall add it.